server.js

```diff
@@ -1,22 +1,21 @@
 (function(global) {
 
 const HERE_FILE = 'here.txt';
```

Comment:
Not sure what you mean here. We're still writing to this file because environment variables can't be written to at runtime like files.